rtypes/item-collection.raml

```diff
@@ -74,6 +74,11 @@
             text/plain:
               example: |
                 "unable to update <<resourcePathName|!singularize>> -- malformed JSON at 13:4"
+        409:
+          description: "Conflict, constraint or optimistic locking failed"
```

Comment:
Under what circumstances (other than optimistic concurrency control, which is new) do FOLIO modules report a 409?

Reply:
I'll try to see the errors that comes out of mod-inventory-storage in unit testing https://issues.folio.org/browse/MODINVSTOR-665

Reply:
As mentioned.. 409 is only thrown for when optimistic locking fails.